torchvision/transforms/functional_tensor.py

```diff
@@ -610,7 +610,7 @@ def affine(
     return _apply_grid_transform(img, grid, interpolation, fill=fill)
 
 
-def _compute_output_size(matrix: List[float], w: int, h: int) -> Tuple[int, int]:
+def _compute_affine_output_size(matrix: List[float], w: int, h: int) -> Tuple[int, int]:
```

Comment:
@datumbox please check this renaming to solve colliding function names and comment out if you approve or disagree. Thanks!

Reply:
Sounds good, thx!